lib/sass_builder.dart

```diff
@@ -56,14 +60,32 @@ class SassBuilder implements Builder {
     var tempAssetPath = tempDir.fileFor(inputId).path;
     _log.fine('compiling file: ${tempAssetPath}');
     var cssOutput = sass.compile(tempAssetPath,
-        packageResolver: new SyncPackageResolver.root(tempDir.packagesDir.uri));
+        packageResolver: new SyncPackageResolver.root(tempDir.packagesDir.uri),
+        style: _getValidOutputStyle());
 
     // Write the builder output
     var outputId = inputId.changeExtension(_outputExtension);
     buildStep.writeAsString(outputId, '${cssOutput}\n');
     _log.fine('wrote css file: ${outputId.path}');
   }
 
+  /// Returns a valid `OutputStyle` value to the `style` argument of [sass.compile] during a [build].
+  ///
+  /// * If [_outputStyle] is not `sass.OutputStyle.compressed` or `sass.OutputStyle.expanded`,
+  ///   a warning will be logged informing the user that the [_defaultOutputStyle] will be used.
+  sass.OutputStyle _getValidOutputStyle() {
+    if (this._outputStyle == sass.OutputStyle.compressed.toString()) {
```

Comment:
Don't use this. when not needed to avoid shadowing.
https://www.dartlang.org/guides/language/effective-dart/usage#dont-use-this-when-not-needed-to-avoid-shadowing